src/main/cpp/subsystems/Elevarm.cpp

```diff
@@ -378,29 +378,21 @@ void Elevarm::assignOutputs()
             wristMotor.setPosition(stowPos.Rotation().Degrees().to<double>());
             previousState.positionState = Position::MANUAL;
         } else {
-
-            // Target in triangle
-            if (futureState.targetPose.theta >= futureState.backMinAngle && futureState.targetPose.theta <= futureState.frontMinAngle){
-                if (futureState.targetPose.h > (carriageMotors.getPosition() - 0.1)){
-                    armRotateMotor.setPosition(futureState.targetPose.theta);
-                }
-                else {
-                    armRotateMotor.setPower(0.0);
-                }
+                    
```

Comment:
Comment indicate this is the "good case" where position is on the same side and no passthrough